### ISSUE TITLE
Problem with `zeitwerk` after update to version 3.4.0

### ISSUE THREAD

**Issue by afuno** [sunny/actor#87]
There seems to be something wrong with `zeitwerk` in release 3.4.0. The application started crashing instantly after updating the gem version from 3.3.0 to 3.4.0.

```
Failure/Error: require File.expand_path("../config/environment", __dir__)

NameError:
  uninitialized constant Actor::ServiceActor

    include ServiceActor::Base
            ^^^^^^^^^^^^
# /usr/local/bundle/gems/service_actor-3.4.0/lib/service_actor.rb:7:in `<class:Actor>'
# /usr/local/bundle/gems/service_actor-3.4.0/lib/service_actor.rb:6:in `<main>'
```

The application code, especially that associated with Actor, has not changed.

Now I'm trying to figure out what's wrong.

**Comment by afuno** [comment]
I still couldn't figure out what was wrong. But the problem is exactly in the changes that are in 3.4.0.

**Comment by afuno** [comment]
@sunny Can you check this issue please? I'm pretty sure it's because of `zeitwerk`, which was introduced in 3.4.0.

All my services are inherited from this class:

```ruby
class ApplicationService
  include ServiceActor::Base
end
```

**Comment by afuno** [comment]
@sunny Fixes: https://github.com/sunny/actor/pull/88

**Comment by sunny** [comment]
Fixed in v3.4.1. This affected apps that required `service_actor/base` instead of `service_actor`. Sorry for that! 🙈
